src/main/resources/db/migration/v0.31.0.4__agm_phenotype_annotation.sql

```diff
@@ -0,0 +1,128 @@
+ALTER TABLE agmphenotypeannotation ADD CONSTRAINT agmphenotypeannotation_inferredallele_id_fk FOREIGN KEY (inferredallele_id) REFERENCES allele (id);
+ALTER TABLE agmphenotypeannotation ADD CONSTRAINT agmphenotypeannotation_inferredgene_id_fk FOREIGN KEY (inferredgene_id) REFERENCES gene (id);
+CREATE INDEX agmphenotypeannotation_phenotypeannotationsubject_index ON agmphenotypeannotation USING btree (phenotypeannotationsubject_id);
+CREATE INDEX agmphenotypeannotation_assertedallele_index ON agmphenotypeannotation USING btree (assertedallele_id);
+CREATE INDEX agmphenotypeannotation_inferredallele_index ON agmphenotypeannotation USING btree (inferredallele_id);
+CREATE INDEX agmphenotypeannotation_inferredgene_index ON agmphenotypeannotation USING btree (inferredgene_id);
+
+CREATE TABLE agmphenotypeannotation_gene (
+	agmphenotypeannotation_id bigint,
+	assertedgenes_id bigint
+);
+
+ALTER TABLE agmphenotypeannotation_gene ADD CONSTRAINT agmphenotypeannotation_gene_agmphenotypeannotation_id_fk FOREIGN KEY (agmphenotypeannotation_id) REFERENCES agmphenotypeannotation (id);
+ALTER TABLE agmphenotypeannotation_gene ADD CONSTRAINT agmphenotypeannotation_gene_assertedgenes_id_fk FOREIGN KEY (assertedgenes_id) REFERENCES gene (id);
+CREATE INDEX agmphenotypeannotation_gene_agmphenotypeannotation_index ON agmphenotypeannotation_gene USING btree (agmphenotypeannotation_id);
+
+CREATE TABLE allelephenotypeannotation (
+	id bigint PRIMARY KEY,
+	phenotypeannotationsubject_id bigint,
+	inferredgene_id bigint
+);
+
+ALTER TABLE allelephenotypeannotation ADD CONSTRAINT agmphenotypeannotation_id_fk FOREIGN KEY (id) REFERENCES phenotypeannotation (id);
+ALTER TABLE allelephenotypeannotation ADD CONSTRAINT agmphenotypeannotation_phenotypeannotationsubject_id_fk FOREIGN KEY (phenotypeannotationsubject_id) REFERENCES allele (id);
+ALTER TABLE allelephenotypeannotation ADD CONSTRAINT agmphenotypeannotation_inferredgene_id_fk FOREIGN KEY (inferredgene_id) REFERENCES gene (id);
+CREATE INDEX allelephenotypeannotation_phenotypeannotationsubject_index ON allelephenotypeannotation USING btree (phenotypeannotationsubject_id);
+CREATE INDEX allelephenotypeannotation_inferredgene_index ON allelephenotypeannotation USING btree (inferredgene_id);
+
+CREATE TABLE allelephenotypeannotation_gene (
+	allelephenotypeannotation_id bigint,
+	assertedgenes_id bigint
+);
+
+ALTER TABLE allelephenotypeannotation_gene ADD CONSTRAINT allelephenotypeannotation_gene_gene_allelephenotypeannotation_id_fk FOREIGN KEY (allelephenotypeannotation_id) REFERENCES allelephenotypeannotation (id);
+ALTER TABLE allelephenotypeannotation_gene ADD CONSTRAINT allelephenotypeannotation_gene_assertedgenes_id_fk FOREIGN KEY (assertedgenes_id) REFERENCES gene (id);
+CREATE INDEX allelephenotypeannotation_gene_allelephenotypeannotation_index ON allelephenotypeannotation_gene USING btree (allelephenotypeannotation_id);
+
```

Comment:
I think there is a missing index on assertedgenes_id?

Reply:
Added now